Release/src/CMakeLists.txt

```diff
@@ -75,6 +75,7 @@ if(CPPREST_EXCLUDE_COMPRESSION)
 else()
   cpprest_find_zlib()
   target_link_libraries(cpprest PRIVATE cpprestsdk_zlib_internal)
+  target_compile_definitions(cpprest PRIVATE -DCPPREST_HTTP_COMPRESSION=1)
```

Comment:
The macro CPPRESET_HTTP_COMPRESSION is an internal macro to http_helpers.cpp, and we shouldn't be messing with it from the build scripts like this.

The comment there indicates that the macro 'CPPREST_EXCLUDE_WEBSOCKETS is a flag that now essentially means "no external dependencies"', and zlib would be such an external dependency.

Perhaps the right thing is to change the bits in http_helpers.cpp to avoid checking CPPREST_EXCLUDE_WEBSOCKETS.